pipelines/zendesk_pipeline.py

```diff
@@ -0,0 +1,15 @@
+import time
+import dlt
+from zendesk import zendesk, zendesk_chat, zendesk_talk
+
+
+if __name__ == "__main__":
```

Comment:
this script is really important. this is the place where people start working with your pipeline.

1. one example where we load all the data is ok
2. demonstrate incremental loads
3. demonstrate pivoting the custom fields
4. if we have any other customizations we must demonstrate them as well